classes/gateways/class.pmprogateway_stripe.php

```diff
@@ -370,10 +373,48 @@ static function pmpro_payment_option_fields( $values, $gateway ) {
 					<?php
 				}
 				?>
-				<input type='hidden' name='stripe_connect_user_id' id='stripe_connect_user_id' value='<?php echo esc_attr( $values['stripe_connect_user_id'] ) ?>'/>
+				<input type='hidden' name='live_stripe_connect_user_id' id='live_stripe_connect_user_id' value='<?php echo esc_attr( $values['live_stripe_connect_user_id'] ) ?>'/>
 				<input type='hidden' name='live_stripe_connect_secretkey' id='live_stripe_connect_secretkey' value='<?php echo esc_attr( $values['live_stripe_connect_secretkey'] ) ?>'/>
-				<input type='hidden' name='test_stripe_connect_secretkey' id='test_stripe_connect_secretkey' value='<?php echo esc_attr( $values['test_stripe_connect_secretkey'] ) ?>'/>
 				<input type='hidden' name='live_stripe_connect_publishablekey' id='live_stripe_connect_publishablekey' value='<?php echo esc_attr( $values['live_stripe_connect_publishablekey'] ) ?>'/>
+			</td>
+        </tr>
+		<tr class="gateway gateway_stripe" <?php if ( $gateway != "stripe" ) { ?>style="display: none;"<?php } ?>>
+            <th scope="row" valign="top">
+                <label> <?php esc_attr_e( 'Stripe Connection (Sandbox):', 'paid-memberships-pro' ); ?></label>
```

Comment:
```suggestion
                <label> <?php esc_html_e( 'Stripe Connection (Sandbox):', 'paid-memberships-pro' ); ?></label>
```

Reply:
Should we include ":" in the translation strings? I think normally that's placed outside of them (after) for maximum flexibility on UI implementation (in case it's removed in the future).

Reply:
For right to left languages, translators need to be able to move the : to the other side. There might be other solutions for handling that.